ee/apps/authorization-service/Dockerfile

```diff
@@ -1,12 +1,74 @@
-FROM node:14.21.3-alpine
+ARG DENO_VERSION="1.37.1"
+
+FROM denoland/deno:alpine-${DENO_VERSION} as deno
+
+FROM node:14.21.3-alpine3.16
 
 ARG SERVICE
 
+## Alpine 3.16 does not have a deno package, but newer versions have it
```

Comment:
if moving the `postintall` script to `build` script as suggested below we probably won't need to do this, since `authorization-service` doesn't uses apps-engine and thus doesn't need `deno`.